README.md

```diff
@@ -1,2 +1,3 @@
 # VSS Website
-The source code for the VSS website.
+The source code for the Virtual Software Streamer (VSS) website.</br>
```

Comment:
Could this be changed to "Virtual Puppet Project (VPupPr)". The name of this repo will be changed accordingly.